.env.base

```diff
@@ -61,6 +61,8 @@ REACT_APP_FEATURE_READ_ONLY_ASSETS=true
 
 # swapper feature flags - other .env files will override these
 REACT_APP_FEATURE_CHAINFLIP=false
+# Specific feature flag for Solana chainflip swaps. If this is disabled, then no swappers support Solana just yet and Solana asset selection should be disabled
+REACT_APP_FEATURE_CHAINFLIP_SOLANA=false
```

Comment:
On second thought, this shouldn't be related to a specific swapper. Should really just be `SWAPPER_SOLANA`, since Jupiter is also hot in the plate.

Reply:
c621add4